packages/pn-personafisica-login/sonar-project.properties

```diff
@@ -1,6 +1,6 @@
 sonar.sources=src
-sonar.exclusions=src/**/__tests__/**, src/**/__test__/**
+sonar.exclusions=src/**/__tests__/**, src/**/__test__/**, src/__mocks__/**, src/utils/onetrust.ts
 sonar.test.inclusions=src/**/__tests__/**, src/**/__test__/**
 sonar.scm.exclusions.disabled=true
 sonar.javascript.lcov.reportPaths=coverage/lcov.info
-sonar.coverage.exclusions=src/models/**, src/locales/**, src/assets/**, src/utils/**, src/**/__tests__/**, src/**/__test__/**
\ No newline at end of file
+sonar.coverage.exclusions=src/models/**, src/locales/**, src/assets/**, src/utils/**, src/**/__tests__/**, src/**/__test__/**, src/__mocks__/**, src/i18n.ts, src/index.tsx, src/utils/onetrust.ts, src/reportWebVitals.ts, src/setupTests.ts
```

Comment:
@AndreaCimini90 You added src/utils/onetrust.ts, but the exclusion rule src/utils/** was already present, which one should we keep?

Reply:
ops i didn't see it

Reply:
Removed the rule src/utils/** beacuse some files are tested